### PR TITLE
Joint deliverables

### PR DESCRIPTION
**Pull request by anssiko**
Related #123

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/das-charter/pull/126.html" title="Last updated on Jun 20, 2023, 4:25 PM UTC (636ce34)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/das-charter/126/05be7da...636ce34.html" title="Last updated on Jun 20, 2023, 4:25 PM UTC (636ce34)">Diff</a>

### CODE REVIEW

**Comment by anssiko** [comment]
This PR is based on the [current DAS WG charter](https://www.w3.org/2022/11/das-wg-charter.html) and is synchronized with the [current Web Apps WG charter](https://www.w3.org/2022/04/webapps-wg-charter.html) and thus adds Contact Picker API as a joint deliverable.

Other joint deliverables may be added subject to bilateral discussions.

See also: [HTML diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2022%2F11%2Fdas-wg-charter.html&doc2=https%3A%2F%2Fdeploy-preview-126--w3c-das-charter.netlify.app%2Fdas-wg-charter) between the current DAS WG charter and this PR.

**Review by tomayac — APPROVED** [review]
Non-owner LGTM

**Review by reillyeon — APPROVED** [review]

File: das-wg-charter.html
Comment (marcoscaceres) [inline das-wg-charter.html]: This is already a fact: 

```suggestion
            <p>
              This work is a joint deliverable with the <a href="https://www.w3.org/groups/wg/webapps">Web Applications Working Group</a>.
            </p>
```

See below from what's on TR:

Screenshot 2023-04-20 at 11 24 49 am

Comment (anssiko) [reply das-wg-charter.html]: Fixed by 3dc341b

File: das-wg-charter.html
Comment (marcoscaceres) [inline das-wg-charter.html]:
```suggestion
              <b>Note:</b> This work is a joint deliverable with the <a href="https://www.w3.org/groups/wg/webapps">Web Applications Working Group</a>.
```

**Comment by marcoscaceres** [comment]
could we make it so that this PR only includes the joint deliverable? It will make it easier to review on our side 🙏

**Comment by reillyeon** [comment]
> could we make it so that this PR only includes the joint deliverable? It will make it easier to review on our side 🙏

+1, moving around Compute Pressure makes the diff harder to read given the HTML preview seems to be broken.

File: das-wg-charter.html
Comment (anssiko) [reply das-wg-charter.html]: Fixed by https://github.com/w3c/das-charter/pull/126/commits/14e2ce982f668c345227154ce06368f6445e8a47

**Comment by anssiko** [comment]
Moved Compute Pressure changes to PR #128, updated this PR accordingly and addressed comments.

@himorin can you please fix the HTML preview and diff.

**Review by reillyeon — APPROVED** [review]
Thanks for the updates. This looks good to me.

**Comment by himorin** [comment]
> @himorin can you please fix the HTML preview and diff.

@anssiko .pr-preview.json itself does not exist... I'll file a PR to setup pr-preview, but was it really worked for this repository!!??

**Review by marcoscaceres — CHANGES_REQUESTED** [review]
This should list: 

 * Geolocation
 * Screen Wake Lock
 * DeviceOrientation Event 

As joint deliverables.

**Comment by marcoscaceres** [comment]
I think we are still waiting on the charter update to list the above joint deliverables with WebApps WG.

**Comment by marcoscaceres** [comment]
Just FYI, here is the proposed Web Apps Charter listing the join deliverables: 
https://w3c.github.io/webappswg/charter/draft-charter-2023.html 

It's expected that the updated charter will go for review to the AC in June, so would be great to have this one up to date also.

**Comment by reillyeon** [comment]
@anssiko, given I just modified this change I will wait for your approval as co-chair before merging.

**Comment by anssiko** [comment]
@reillyeon, https://github.com/w3c/das-charter/pull/126/commits/bd57663b4e584e1a3475bad81ee935f64d57ff18 and https://w3c.github.io/webappswg/charter/draft-charter-2023.html refer to a different Geolocation spec.

I believe WebApps WG's aspiration is to help with the maintenance of the Geolocation API that has reached the W3C Recommendation status (and is revised per https://www.w3.org/2023/Process-20230612/#revising-rec). Recommendations are listed under the Maintenance section of the charter.

**Comment by reillyeon** [comment]
> @reillyeon, [bd57663](https://github.com/w3c/das-charter/commit/bd57663b4e584e1a3475bad81ee935f64d57ff18) and https://w3c.github.io/webappswg/charter/draft-charter-2023.html refer to a different Geolocation spec.

Oops, fixed.

**Pull request merged**
